test/quantization/test_quantized_op.py

```diff
@@ -2722,7 +2721,7 @@ def _test_embedding_bag_unpack_fn(self, pack_fn, unpack_fn, num_embeddings, embe
         weights = torch.from_numpy((np.random.random_sample((
             num_embeddings, embedding_dim)) + 1).astype(np.float32))
 
-        w_packed = pack_fn(weights)
+        w_packed = pack_fn(weights, optimized_qparams=True)
```

Comment:
Should we test for both cases of optimized_qparams for completeness?

Reply:
Yes we should, not sure how I missed this.